.github/workflows/documentation.yaml

```diff
@@ -27,6 +27,8 @@ jobs:
     steps:
 
       - uses: actions/checkout@v2.3.4
+        with:
```

Comment:
does this do?

Reply:
This should be fine, it maybe a good idea to upgrade to `actions/checkout@v3` (cf https://github.com/actions/checkout)

Reply:
done